docs/other-topics/utility-types.md

```diff
@@ -0,0 +1,98 @@
+# Utility TypeScript Types
+
+## Typing a Model Class
+
+[`ModelStatic`](pathname:///api/v7/index.html#ModelStatic) is designed to be used to type a Model *class*.
+
+Here is an example of a utility method that requests a Model Class, and returns the list of primary keys defined in that class:
+
+```typescript
```

Comment:
Can we automatically wrap this? Since the imports are too long

Reply:
If it were in a codebase sure, but that's not the important part of the example so I would rather keep it all on one line until we have a way to collapsing the unimportant sections (A bit like MUI's documentation)